src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -1031,6 +1028,31 @@ public void Sort(IComparer<T> comparer)
             return true;
         }
 
+        private void AddEnumerable(IEnumerable<T> enumerable)
+        {
+            Contract.Assert(enumerable != null);
+            Contract.Assert(!(enumerable is ICollection<T>), "We should have optimized for this beforehand.");
+
+            using (IEnumerator<T> en = enumerable.GetEnumerator())
+            {
+                _version++; // Even if the enumerable has no items, we can update _version.
```

Comment:
`AddRange` already increments `_version`. Can we remove the increment here?

The other caller of this method is the .ctor, which shouldn't need the `_version` increment.